### ISSUE TITLE
[google_maps_flutter_web] Add "draggableCursor" option

### ISSUE THREAD

**Issue by F1ix1337** [flutter/flutter#68297]
It would be really nice if the "draggableCursor" option could be added to change the cursor style. By default its the hand icon.

**Comment by yjbanov** [comment]
Agreed that this would be a nice feature. Would you consider contributing this feature to the plugin?
